build.gradle

```diff
@@ -1,16 +1,20 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.1.2'
     }
 }
 
 apply plugin: 'com.android.library'
 
-repositories {
-    jcenter()
+allprojects {
+    repositories {
+        google()
+        jcenter()
+    }
 }
 
 dependencies {
```

Comment:
please change 'compile' to 'implementation'.

Reply:
`implementation` is working everywhere except for these cases, it seems that the dependencies are not found and I see the next errors when using implementation and executing `gradlew clean build`

`implementation 'org.apache.jackrabbit:jackrabbit-webdav:2.12.4'`=> Cannot access HttpClient in sampleclient/MainActivity.java

`implementation 'com.squareup.okhttp3:okhttp:3.10.0'`=> package okhttp3 does not exist

Reply:
Hmmm, we should use api instead of compile.